src/components/pages/Projects.js

```diff
@@ -1,12 +1,99 @@
+                    Using our 6 available printers, we aim to offer low-cost, high quality 3D 
+                    printing services to UBC students, staff, and faculty. <br /><br />
+                    Recently last year, our brand new Consulting team has also  kicked off, 
+                    now providing the first CAD modelling and design consultation service on campus! 
+                </p>
+                <center><img src="/images/projects1.png" alt=""></img></center>
+                <br /><br />
+                <h5>3D PRINTER DESIGN AND CONSTRUCTION</h5>
+                <p>We  enjoy experimenting and pushing the boundaries of what 3D printers can  
+                    do, optimizing them to work faster, smarter and add clever features.  
+                    This year, we have launched our newest and most ambitious project to date: 
+                    To build UBC Rapid's first original compact 3D printer, ideal for engineering 
+                    students with limited space or hobbyists that would like to travel with their 
+                    printer. The printer is still in its early development stages. Some other
+                    Printer development ideas considered include a Monorail printer, and a Cable 
+                    driven printer. 
+                </p>
+                <center><img src="/images/projects2.png" alt=""></img></center>
+                <br /><br />
+                <h5>EXOTIC MATERIAL TESTING</h5>
+                <p>UBC Rapid will be expanding the variety of materials available for 
+                    student and faculty rapid prototyping projects. Currently, most printers 
+                    available to UBC undergraduate students are limited to either ABS and PLA. 
+                    While both materials are useful in prototyping, they severely limit the 
+                    types of parts students can produce. The biggest challenge is the lack 
+                    of generally accepted guidelines for printing in materials other than 
+                    PLA and ABS; this project aims to fill in the gaps of knowledge by 
+                    developing resources for using non-standard materials on 3D printers, 
+                    and ultimately make other materials (such as composites) available for 
+                    student/faculty use. Filaments to be tested include:
+                </p>
+                <p>- Metal (Aluminum, Brass, Bronze, Stainless Steel) <br /><br />
+                    - Carbon fiber <br /><br />
+                    - Flexible (TPU, TPE) <br /><br />
+                    - Polycarbonate <br /><br />
+                    - PTFE (Nylon) <br /><br />
+                    - Water soluble <br /><br />
+                    - PETG <br /><br />
+                    - Conductive <br /><br />
+                    - Magnetic <br /><br />
+                    - Wax and many more!
```

Comment:
To create lists in html, use ul (unordered lists) for bullet points, and ol (ordered lists) for numbered lists. See here for more information: https://www.w3schools.com/html/html_lists.asp